### ISSUE TITLE
Annotations in description

### ISSUE THREAD

**Issue by sourishkrout** [stateful/runme#334]
We probably want to skip the fenced code block metadata for the purposes of using it as description (if possible).

```
NAME	FILE	FIRST COMMAND	DESCRIPTION
relative-pwd	cwd/RELATIVE.md	pwd	cwd: ../../...
relative-rel-pwd	cwd/RELATIVE.md	pwd	
relative-abs-pwd	cwd/RELATIVE.md	pwd	
none-pwd	cwd/NONE.md	pwd	
none-rel-pwd	cwd/NONE.md	pwd	
none-abs-pwd	cwd/NONE.md	pwd	
absolute-pwd	cwd/ABSOLUTE.md	pwd	cwd: /tmp
absolute-rel-pwd	cwd/ABSOLUTE.md	pwd	
absolute-abs-pwd	cwd/ABSOLUTE.md	pwd	
```

**Comment by sourishkrout** [comment]
Not a high priority. Whenever you have a moment to squeeze it in @mxsdev.

**Comment by mxsdev** [comment]
@sourishkrout 

Fixed in #335
